test/Tizen.NUI.Scene3D.Sample/Scene3DSample.cs

```diff
@@ -452,7 +459,7 @@ void OnKeyEvent(object source, Window.KeyEventArgs e)
                 }
                 case "f":
                 {
-                    if (mModelAnimation?.State == Animation.States.Playing)
+                    if (mModelAnimation != null && mModelAnimation?.State == Animation.States.Playing)
                     {
                         if (mModel != null && mModelLoadFinished)
```

Comment:
This can be modified to check mModelLoadFinished only.

Reply:
Thanks! I remove '?' keyword. Actually NUI.BaseHandle override the == operator. And that operation is stronger than simple ? operation. So I choose it.

Reply:
same mistake like above... sorry T.T